OK, it looks like we were able to get the secrets lined up and build a new docs-builder image. However, because you opened this PR from a fork, the GitHub App couldn't update your PR. We generally expect someone from docs-structure to manage this by pushing a branch into the cilium/cilium org.

Here's the error message. If you manually apply this diff to your PR and re-push, I think it should work:

```
diff --git a/.github/workflows/documentation.yaml b/.github/workflows/documentation.yaml
index 6a9344e8..d33ce100 100644
--- a/.github/workflows/documentation.yaml
+++ b/.github/workflows/documentation.yaml
@@ -64,7 +64,7 @@ jobs:
           # Needed to detect missing redirects
           fetch-depth: 0
       - name: Build HTML
-        uses: docker://quay.io/cilium/docs-builder:550ca417f5dc0b13fa7f117cc117443b[23](https://github.com/cilium/cilium/actions/runs/12950641283/job/36303165207#step:9:24)120f5e@sha256:febb1339085d9bcabcfd547ab2125e67e1c6468f0bc33f62f9326e790efb938a
+        uses: docker://quay.io/cilium/docs-builder:[24](https://github.com/cilium/cilium/actions/runs/12950641283/job/36303165207#step:9:25)77f7372c2e92fa217b786bd60e705abb3ef03e@sha256:628a512cbd5e28bc8eb323d2ede49917c8cba40e859a484b828d956854dbcd94
         with:
           entrypoint: ./Documentation/check-build.sh
           args: html
To https://github.com/paularah/cilium.git
 ! [remote rejected]   HEAD -> pr/paularah/slack_url (refusing to allow a GitHub App to create or update workflow `.github/workflows/documentation.yaml` without `workflows` permission)
error: failed to push some refs to 'https://github.com/paularah/cilium.git'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/documentation.yaml
+++ b/.github/workflows/documentation.yaml
@@ -64,7 +64,7 @@ jobs:
           # Needed to detect missing redirects
           fetch-depth: 0
       - name: Build HTML
-        uses: docker://quay.io/cilium/docs-builder:550ca417f5dc0b13fa7f117cc117443b23120f5e@sha256:febb1339085d9bcabcfd547ab2125e67e1c6468f0bc33f62f9326e790efb938a
+        uses: docker://quay.io/cilium/docs-builder:5f497e3f4c73f217a780c8038f9174ceabfabe57@sha256:37105a84068e427116bad64f8812ef5f4239812fd4f085252901b4c7cddb48af
         with:
           entrypoint: ./Documentation/check-build.sh
           args: html

--- a/Documentation/requirements.txt
+++ b/Documentation/requirements.txt
@@ -3,7 +3,7 @@ Sphinx==7.1.2
 sphinx-autobuild==2021.3.14
 
 # Custom theme, forked from Read the Docs
-sphinx-rtd-theme-cilium @ git+https://github.com/cilium/sphinx_rtd_theme.git@84a34e7f301e9bfed5330da9acdb65d62020af4d
+sphinx-rtd-theme-cilium @ git+https://github.com/cilium/sphinx_rtd_theme.git@3047cc4ad7aca4ed6184b6b40658910af2130f4d
 # We use semver to parse Cilium's version in the config file
 semver==3.0.1
 # Sphinx extensions

